eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/test/Azure.Generator.Mgmt.Tests/Providers/TestData/ManagementLongRunningOperationProviderTests/Verify_Generic_LROProviderGeneration.cs

```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
```

Comment:
We should add the license config in the emitter.

Reply:
added license info back